docs/releases/upgrading/upgrade-v6.0.md

```diff
@@ -1,13 +1,157 @@
+The steps listed in this article require an existing local installation of
+InvenioRDM 4.0, please make sure that this is given! If unsure, run
+`invenio-cli install` from inside the instance directory before executing the
+listed steps.
 
-**Note**: Do *not* delete the old Python virtual environment, or the database migration may complain about missing packages.
+**Note**: Do *not* delete the old Python virtual environment, or the database
+migration may complain about missing packages.
 
 !!! warning "Backup"
 
     Always backup your database and files before you try to perform an upgrade.
+
+## Upgrade steps
+
+### Upgrade your instance dependencies
+
+Bump the RDM version and rebuild the assets:
+
+```bash
+invenio-cli packages update 6.0.0
+invenio-cli assets build -d
+```
+
+### Prepare vocabularies
+
+Before migrating your data, we have to check that it is compatible with the
+v6.0 schemas. Note that many changes have been made to enable the use of
+vocabularies.
+
+First of all we have to upgrade the DB:
+
+```
+invenio alembic upgrade
+```
+
+Then, we need to fix the previous version vocabularies. For that we are going
+to run the migration script and choose option number 1. It is **very important
+to follow the steps in order**.
+
+```bash
+invenio shell $(find $(pipenv --venv)/lib/*/site-packages/invenio_app_rdm -name migrate_4_0_to_6_0.py)
+
+Choose a step:
+[1] Migrate old vocabularies
+>> 1
+```
+
+Another two important new vocabularies are affiliations and subjects. For this
+we need to run specific checks:
+
+**Subjects**
+
+```bash
+invenio shell $(find $(pipenv --venv)/lib/*/site-packages/invenio_app_rdm -name migrate_4_0_to_6_0.py)
+
+Choose a step:
+[2] Check record\'s subjects
+>> 2
+```
+
+This script will check that your subjects are either free text, which requires
+no action on your side, or that they should be a custom vocabulary. In the
+latter case, a yaml file will be created in your current folder (`custom_subjects.yaml`).
+
+You move this file to the `app_data` folder and create a `vocabularies.yaml`
+file. Then you must reference the subjects file, for example:
+
+```yaml
+subjects:
+  pid-type: sub
+  schemes:
+    - id: Custom
+      name: Custom subjects
+      data-file: custom_subjects.yaml
+```
+
+The content of this file will be added to your instance's subjects vocabulary
+when running the `fixtures` command.
+
+**Affiliations**
+
+```bash
+invenio shell $(find $(pipenv --venv)/lib/*/site-packages/invenio_app_rdm -name migrate_4_0_to_6_0.py)
+
+Choose a step:
+[3] Check creators/contributors affiliations
+>> 3
+```
+
+This script will check that your creators and contributors affiliations are
+either free text, which will require no action on your side, contain a ROR
+identifier or that they should be a custom vocabulary.
+
+If the affiliations contain ROR identifiers, you will need to add that 
+vocabulary. See more details [here](TODO). Otherwise, you will need to create a
```

Comment:
Note not to forget to add this!

Reply:
Fixed: won't work until https://github.com/inveniosoftware/docs-invenio-rdm/pull/207 is merged but the location has been agreed.